src/MultiSelect.js

```diff
@@ -1,51 +1,63 @@
+// @flow
 import React from 'react';
 
-class MultiSelectOption extends React.Component {
+type MultiSelectOptionProps = {
+    label: string,
+    value: string
+};
+
+class MultiSelectOption extends React.Component<MultiSelectOptionProps, {}> {
     render() {
         return(
             <option value={this.props.value}>{this.props.label}</option>
         );
     }
 }
 
-export default class MultiSelect extends React.Component {
-    constructor(props) {
+type OptionDef = {
+    value: string,
+    label?: string
+}
+
+type MultiSelectProps = {
+    title: string,
+    selectedValues?: Array<Object>,
```

Comment:
'Object' is an alias to 'any' and will be deprecated and removed in a future version of Flow -- https://flow.org/en/docs/types/objects/ at the bottom of the page

Reply:
Good to know, I am opposed in general to using `any`, `Object`, or `Function` if we can help it.  I removed nearly all of them on the latest commit (still reading up on refs).